### DISCUSSION TITLE
No result for search term which only exists in one specific managed property

### DISCUSSION THREAD

**Discussion by WinOnMarco** [microsoft-search/pnp-modern-search#3787] [Q&A] [answered]
We've set up a search page which is searching documents from one specific document library that consists of several different properties (metadata columns like "Document No" (Single line of text), "Info" (single line of text), ...). Overall this search page works fine. The refiners work fine for all required managed properties (we've defined some RefinableStrings).

But if we search for a specific term (e.g. SLS) which only exists in the "Info" property (Crawled prop. 'ows_info' which is mapped as 'RefinableString109'), we get no results.

If I search by explicitly by targeting the managed property (like "RefinableString109:SLS") we get the expected result. Thus, the index seems to be ok.

We've then checked the following: The property "RefinableString109" is selected in the search result web part under "selected properties".

Does anybody have an idea why we don't get the result without explicitly searching for the refinablestring109?

Or any hints how we can troubleshoot this?

**Comment by wobba** [comment, marked as answer]
Here's your answer :) https://www.techmikael.com/2020/03/side-effect-of-mapping-crawled.html

**Comment by WinOnMarco** [comment]
That was it. Thanks a lot.
